site/en/blog/new-in-chrome-96/index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: New in Chrome 96
+description: >
+  Chrome 96 is rolling out now! There are two new properties in the web app
+  manifest. The id property allows you to specify a unique ID for your PWA,
+  and the protocol_handlers property allows you to automatically register your
+  PWA as a protocol handler upon installation. There's a new origin trial that
+  allows you to specify priority hints for downloading resources. And there's
+  plenty more!
+layout: 'layouts/blog-post.njk'
+date: 2021-11-16
+authors:
+  - petelepage
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/gyhkrKnrs5TxwcF0Ybke.jpg'
+alt: >
+  New in Chrome hero logo
+tags:
+  - new-in-chrome
+  - chrome-96
+---
+
+{% YouTube id='TODO' %}
+
+Here's what you need to know:
+
+* The [manifest `id` property](#manifest-id) lets you specify a unique ID for
+  your PWA.
+* The [`protocol_handlers` property](#pwa-protocol-handlers) allows you to
+  automatically register your PWA as a protocol handler upon installation.
+* The [priority hints origin trial](#pri-hints) allows you to specify
+  priority hints for downloading resources.
+* You can [force the Chrome version to 100](#chrome-100) to test how your code
+  will react to a three digit version number.
+* Videos from the [Chrome Dev Summit](#cds-videos) are all online.
+* And there's plenty [more](#more).
+
+I'm [Pete LePage](https://petelepage.com), finally shooting
+**in the studio**, let's dive in and see what's new for developers in Chrome 96.
+
+## Manifest `id` for PWAs {: #manifest-id }
+
+When a user installs a PWA, the browser needs a way to uniquely identify it.
+But, until recently, the web app manifest spec didn't specify how to identify
+a PWA, leaving browsers to decide, and leading to different implementations.
+In some browsers, the `start_url` is used, while in others, the path to the
+manifest file is used.
+
+That makes it impossible to change either of those fields without breaking the
+install experience! Now, there's a new optional `id` property, that allows
```

Comment:
```suggestion
install experience. Now, there's a new optional `id` property, that allows
```

Reply:
Fixed in 1475b8e031abe034773b744274e64066738153ad